### PR TITLE
Global help button

### PR DESCRIPTION
**Pull request by Piotrek605**

### CODE REVIEW

File: meteor-app/imports/ui/layouts/full-window/style.less [hunk 1]
```diff
@@ -10,30 +10,25 @@
   align-items: stretch;
 
   .page__header {
-    flex: 0 0 auto;
```
Comment (Zodiase) [inline meteor-app/imports/ui/layouts/full-window/style.less]: Why remove this line?

File: meteor-app/imports/ui/layouts/full-window/style.less [hunk 2]
```diff
@@ -10,30 +10,25 @@
...
-  }
-
-  .page__body {
-    flex: 1 1 0;
```
Comment (Zodiase) [inline meteor-app/imports/ui/layouts/full-window/style.less]: Why remove this line?

File: meteor-app/imports/ui/layouts/full-window/style.less [hunk 3]
```diff
@@ -10,30 +10,25 @@
...
-  }
-
-  .page__footer {
-    flex: 0 0 auto;
```
Comment (Zodiase) [inline meteor-app/imports/ui/layouts/full-window/style.less]: Why remove this line?

File: meteor-app/imports/ui/layouts/full-window/style.less [hunk 4]
```diff
@@ -10,30 +10,25 @@
...
+    .flex-row;
 
-  .page__header {
     padding: @global-layout-padding;
     margin-bottom: @global-layout-margin;
     background-color: white;
   }
 
   .page__body {
+    flex: 1 1 0;
```
Comment (Zodiase) [inline meteor-app/imports/ui/layouts/full-window/style.less]: The flexbox config was split away since it is only relevant to the parent-children relationships. Puting it inside here makes the code harder to port.

File: meteor-app/imports/ui/layouts/main/component.js [hunk 5]
```diff
@@ -11,19 +11,22 @@ export default class MainLayout extends React.Component {
     const {
       store,
       body,
+      helpUrl,
     } = this.props;
+
     return (
       <div className="page layout-main">
         <div className="page__header">
-          <p>Some header</p>
```
Comment (Zodiase) [inline meteor-app/imports/ui/layouts/main/component.js]: Why is this change necessary to this PR?

**Comment by Piotrek605** [comment]
Well that's why you should specify a min width and height as I said.

File: meteor-app/imports/ui/layouts/main/component.js (hunk 5 above)
Comment (Piotrek605) [inline meteor-app/imports/ui/layouts/main/component.js]: So that you can see and adjust the new layout of the header and footer without having unnecessary things there.

**Comment by Zodiase** [comment]
Nope the cause of the bug is the removal of `flex: 0 0 auto`.

File: meteor-app/imports/ui/layouts/main/component.js (hunk 5 above)
Comment (Zodiase) [inline meteor-app/imports/ui/layouts/main/component.js]: It's not unnecessary. It's used for taking up space for potential content so the header doesn't look absurdly narrow, especially on home page, which uses `main` layout.

See how home page looks like with this change:
image
